pkg/logging/logging.go

```diff
@@ -0,0 +1,108 @@
+// package logging is a small wrapper around github.com/k8snetworkplumbingwg/cni-log
+
+package logging
+
+import (
+	cnilog "github.com/k8snetworkplumbingwg/cni-log"
+)
+
+const (
+	labelCNIName     = "cniName"
+	labelContainerID = "containerID"
+	labelNetNS       = "netns"
+	labelIFName      = "ifname"
+	cniName          = "sriov-cni"
+)
+
+var (
+	logLevelDefault = cnilog.InfoLevel
+	containerID     = ""
+	netNS           = ""
+	ifName          = ""
+)
+
+// Init initializes logging with the requested parameters in this order: log level, log file, container ID,
+// network namespace and interface name.
+func Init(logLevel, logFile, containerIdentification, networkNamespace, interfaceName string) {
+	setLogLevel(logLevel)
+	setLogFile(logFile)
+	containerID = containerIdentification
+	netNS = networkNamespace
+	ifName = interfaceName
+}
+
+// setLogLevel sets the log level to either verbose, debug, info, warn, error or panic. If an invalid string is
+// provided, it uses error.
+func setLogLevel(l string) {
+	ll := cnilog.StringToLevel(l)
+	if ll == cnilog.InvalidLevel {
+		ll = logLevelDefault
+	}
+	cnilog.SetLogLevel(ll)
+}
+
+// setLogFile sets the log file for logging. If the empty string is provided, it uses stderr.
+func setLogFile(fileName string) {
+	if fileName == "" {
+		cnilog.SetLogStderr(true)
+		cnilog.SetLogFile("")
+		return
+	}
+	cnilog.SetLogFile(fileName)
+	cnilog.SetLogStderr(false)
+}
+
+// SetContainerID sets the ContainerID to prepend to every log message.
+func SetContainerID(cID string) {
```

Comment:
nit: This function is not used in this project, and I don't see any specific value in exporting it as public. Can we get rid of it?

Same for  `SetNetNS` and `SetIFName`?